shared/test/otSdkDriver_test.js

```diff
@@ -157,6 +157,26 @@ describe("loop.OTSdkDriver", function() {
         sinon.match.instanceOf(HTMLDivElement),
         expectedConfig);
     });
+
+    it("should call onPublishComplete", function() {
```

Comment:
Thinking about it, we probably don't need to test for calling onPublishComplete directly. I'd expect tests here for:

`"should not do anything if publishing was successful"`
`"should clean up the publisher if there was an error"`
`"should dispatch a ConnectionFailure if there was an error"`
`"should notify metrics if there was an error"`